pkg/kernel/kernel_config-5.10.x-x86_64

```diff
@@ -2186,7 +2186,8 @@ CONFIG_MII=m
 CONFIG_NET_CORE=y
 CONFIG_BONDING=m
 CONFIG_DUMMY=m
-# CONFIG_WIREGUARD is not set
```

Comment:
Out of curiosity. You just did 'make menuconfig' and enabled `CONFIG_WIREGUARD`. And all the other changes are done by linux's configuration utility? Or there is more to that.

The reason I ask, is that I had an urge to introduce config fragments for a little while. Which is a bit hard to do at the moment without knowing the intention of changes which have been done to `.config` in the past.